src/ledger/assets/tokens.ts

```diff
@@ -20,8 +20,18 @@ export class Tokens extends Asset {
 
 	constructor(v: bigint[]) {
 		super();
+		Tokens.requireUniquePositiveEntries(v);
 		this.value = v;
 	}
```

Comment:
If our data model _requires_ `v` to be sorted, because the class's implementations don't work otherwise, shouldn't we sort it in the constructor? I'd create a sorted copy of `v` to `this.value` and then call `requireUniquePositiveEntries` on it.

Reply:
Agree.

Reply:
ok!